.github/workflows/release.yml

```diff
@@ -1,30 +1,34 @@
-name: release
+name: Generate Next Release
 
 on:
-  push:
-    tags:
-    - '*'
+  workflow_dispatch:
 
 jobs:
-  goreleaser:
+  release:
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      -
-        name: Set up Go
+      - name: Set up Go
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
-      -
-        name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v5
+      - name: Release
+        id: semantic-release
+        uses: cycjimmy/semantic-release-action@v4
         with:
-          version: latest
-          args: release --rm-dist
+          semantic_version: 19.0.5
```

Comment:
what's the reason to keep 19.0.5? https://github.com/semantic-release/semantic-release/releases/tag/v24.0.0

Reply:
There's no reason any more.  I bumped this version and the conventional commits changelog plugin version.